HelpMyStreetFE/HelpMyStreetFE/Views/Shared/DisplayTemplates/RequestSummaryViewModel.cshtml

```diff
@@ -129,15 +147,12 @@
                 }
             }
             <span class="job__info__activity__location">
-                @if (Model.Location != null)
-                {
-                    @Model.Location.LocationDetails.Name
-                }
-                else
-                {
-                    @Model.Item.PostCode
-                }
+                [@requestHMSReference]
+            </span>
+            <span class="job__info__activity__location">
+                @location
```

Comment:
We need to keep showing `@Model.Location.LocationDetails.Name` where there is a Location.  (This shows the vaccine centre name for shifts.)